### ISSUE TITLE
Keyboard focus order in expandable section at bottom of sidebar does not allow access to main content of page; 

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#844]
**Describe the bug**

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html 
2. Click on the expandable section at bottom of sidebar to expand it, to work around the bug regarding lack of keyboard access to this section
3. Click within the expanded section and then press TAB to get focus on first link in that section
4. Press TAB repeatedly to navigate between links

Notice: After links, keyboard focus continues out of page and to rest of browser

**Expected behavior**
Keyboard focus to proceed from sidebar to main page content

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/c33be98b-5b50-4815-bc47-ce666f33e287)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.
